packages/api/src/external/carequality/patient.ts

```diff
@@ -56,23 +94,72 @@ export async function discover(
   }
 }
 
-export async function handleDiscoverResponse(
-  patientDiscoveryResponse: PatientDiscoveryResponse
+export async function handlePatientDiscoveryResults(
+  patient: Patient,
+  pdResults: PatientDiscoveryResult[]
 ): Promise<void> {
-  const { cxId, patientId, xcpdPatientId, patientMatch, gateway } = patientDiscoveryResponse;
-
-  if (patientMatch) {
-    await setCarequalityId({
-      patientId,
-      cxId,
-      carequalityPatientId: xcpdPatientId?.id ?? "",
-      carequalityPatientSystemId: xcpdPatientId?.system ?? "",
-    });
-  } else {
-    // WE SHOULD EXPECT THIS TO HAPPEN SO SHOULD WE ADD SENTRY MESSAGE?
-    // DONT WANT TO BOMBARD ALERTS CHANNEL
-    console.log(
-      `No patient match in IHE Gatway for patientId ${patientId}, cxId ${cxId}, gatewayUrl ${gateway.url}, gatewayId ${gateway.oid}`
-    );
+  if (pdResults.length === 0) {
+    console.log(`No patient discovery results found for patientId: ${patient.id}`);
+    return;
   }
+  const { id, cxId } = patient;
+
+  const cqLinks = buildCQLinks(pdResults);
+  const pdCreate: PatientCQDataCreate = {
+    id,
+    cxId,
+    data: { links: cqLinks },
+  };
```

Comment:
super nit: this doesn't need to execute if `cqLinks` is empty, so could just set this as parameter of `createOrUpdatePatientCQData` w/o creating a const.